### PR TITLE
Rzr 8 header 

### PR DESCRIPTION
**Pull request by SashaGraves**

### CODE REVIEW

File: source/shared/tel-number.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import '../../scss/tel-number.scss';
+
+function TelNumber() {
```
Comment (cashidze) [inline source/shared/tel-number.tsx]: Rename: PhoneNumber (rename file accordingly)

Comment (cashidze) [reply source/shared/tel-number.tsx]: Can we replace it with arrow function?

File: source/shared/logo.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import image from '../../images/false-logo.png';
+
+function Logo() {
```
Comment (cashidze) [inline source/shared/logo.tsx]: Can we replace it with arrow-function?

File: source/shared/call-me-button.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import '../../scss/call-me-button.scss';
+
+function CallMeButton() {
```
Comment (cashidze) [inline source/shared/call-me-button.tsx]: Cal we replace it with arrow function?

File: source/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import SmallHeader from './call-me-header';
+import NavHeader from './nav-header';
+
+function Header() {
```
Comment (cashidze) [inline source/header/header.tsx]: Try to replace with arrow function
Try to replace function ...   export defaut ... with one call

smth like this:
export * Header => ()

File: source/header/call-me-header.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import TelNumber from '../shared/tel-number';
+import CallMeButton from '../shared/call-me-button';
+
+function SmallHeader() {
```
Comment (cashidze) [inline source/header/call-me-header.tsx]: try replace with export * () => {}

File: scss/custom.scss [hunk 6]
```diff
@@ -1,3 +1,24 @@
+/*DEVELOPE*/
```
Comment (cashidze) [inline scss/custom.scss]: DEVELOP

File: source/header/call-me-header.tsx (hunk 5 above)
Comment (cashidze) [reply source/header/call-me-header.tsx]: incosistent naming. filename should match with export default function

File: source/shared/call-me-button.tsx (hunk 3 above)
Comment (SashaGraves) [reply source/shared/call-me-button.tsx]: yes

File: scss/custom.scss (hunk 6 above)
Comment (SashaGraves) [reply scss/custom.scss]: ok

File: source/shared/tel-number.tsx (hunk 1 above)
Comment (SashaGraves) [reply source/shared/tel-number.tsx]: yes

File: source/shared/logo.tsx (hunk 2 above)
Comment (SashaGraves) [reply source/shared/logo.tsx]: yes

File: source/header/header.tsx (hunk 4 above)
Comment (SashaGraves) [reply source/header/header.tsx]: yes to arrow
no to one-line-export-and-declaration, sorry, i like it this way so far

File: source/header/call-me-header.tsx (hunk 5 above)
Comment (SashaGraves) [reply source/header/call-me-header.tsx]: ok

**Pull request merged**